console/client/src/features/timeline/TimelineFilterBar.tsx

```diff
@@ -24,7 +24,7 @@ export const TimelineFilterBar = ({ selectedFilters, onFilterChange }) => {
   return (
     <>
       <div className={`sticky top-0 z-10 ${panelColor} shadow`}>
-        <div className='flex items-center justify-between p-4'>
+        <div className='flex items-center justify-between px-4'>
```

Comment:
I think we want to keep this `p-4` so there's padding on the top and bottom of the filter bar as well